jaeger-spark-dependencies-cassandra/src/test/java/io/jaegertracing/spark/dependencies/cassandra/CassandraDependenciesDockerJobTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017 The Jaeger Authors
```

Comment:
This should better be 2025, however, the current build script can't handle it as it expects the same date for every file which is invalid.
I've filed it as https://github.com/jaegertracing/spark-dependencies/issues/151